po/zh_CN.po

```diff
@@ -184,30 +184,33 @@ msgid ""
 "to a negative value, `SBD_WATCHDOG_TIMEOUT` must be set to the same value on "
 "all nodes that use SBD, otherwise data corruption or loss could occur."
 msgstr ""
-"如果设置为正值，则假定丢失的节点在这段时间内使用基于watchdog的SBD进行自我防"
-"护。这不需要明确配置fence资源，但可以配置一个fence_watchdog资源，以限制特定节"
-"点的使用。如果设置为0（默认值），集群将永远不会假定基于watchdog的自我防护。如"
-"果设置为负值，且如果`SBD_WATCHDOG_TIMEOUT`环境变量的本地值为正值，则集群将使"
-"用该值的两倍，否则将其视为0。警告：在使用基于watchdog的SBD的所有节点上，此超"
-"时必须大于`SBD_WATCGDOG_TIMEOUT`,如果本地值不是这样，或者SBD未运行，则"
-"Pacemaker将拒绝在任何节点上启动。如果设置为负值，则在使用SBD的所有节点上，"
-"`SBD_WATCHDOG_TIMEOUT`必须设置为相同的值，否则可能会发生数据损坏或丢失。"
+"这个值如果设置为正值, 丢失的节点将在设定的时间内将被假设使用基于 "
+"watchdog (watchdog-based)的SBD进行自我 fence (self-fencing)."
+"这不需要明确配置一个隔离(fence)资源, 但可以配置一个 fence_watchdog "
```

Comment:
So far "fence" has remained literal without being translated everywhere. Probably we should keep it consistent here as well.